doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -727,6 +727,30 @@ declare your constant as rigid for proof search using the command
 Strategies for rewriting
 ------------------------
 
+Usage
+~~~~~
+
+.. tacn:: rewrite_strat @rewstrategy {? in @ident }
+   :name: rewrite_strat
+
+   Rewrite using the strategy s in hypothesis ident or the conclusion.
+
+   .. exn:: Nothing to rewrite.
+
+      If the strategy failed.
```

Comment:
```suggestion
      If the strategy failed to find any matches.
```

(There are other kinds of failures, like "No constraint can apply on a dependent ..." and type errors (which are generally bugs.)